compiler/rustc_span/src/symbol.rs

```diff
@@ -591,6 +591,7 @@ symbols! {
         cfg_accessible,
         cfg_attr,
         cfg_attr_multi,
+        cfg_attr_trace: "cfg_attr-trace", // must not be a valid identifier
```

Comment:
According to the comment at the top of this list, this should be called `cfg_attr_dash_trace`, similar to `proc_dash_macro` for the symbol `proc-macro`.

Reply:
It can be renamed to `<cfg_attr>` or something else, because the specific string value is not important, but I'd rather keep the `sym::cfg_attr_trace` naming.